developing.adoc

```diff
@@ -46,6 +46,34 @@ For more information, see the following:
 . link:https://github.com/golang/go/wiki/Modules[Introduction to Go modules]
 . link:https://blog.golang.org/using-go-modules[Using Go modules]
 
+[[compiling-binaries]]
+== Compiling the CRC Binary
+
+Run the following Makefile target that will produce binaries for multiple platforms:
+
+----
+$ make cross
+----
+
+In order to compile the crc binary for a specific platform, run the following command:
+
+----
+$ make out/linux-amd64/crc
```

Comment:
This can be replaced with `make` because we have `default` target in the Makefile is `install` which will build the binary on `$GOPATH/bin` and also platform independent.

Reply:
For the above suggestion, the statement would also need to be changed to something like `In order to compile the crc binary for your platform, and install it in $GOBIN`...

Reply:
That sounds reasonable to me. Any suggestions on how we'd document, or whether we see value in documenting, how to install CRC or the embedded binaries for a specific platform? It seems a bit awkward to me in the current form, but maybe that's because we'd have to specify an override variable if we're interested in overriding the default binary output path to something more traditional like `bin` for Go-based projects.

Reply:
> It seems a bit awkward to me in the current form, but maybe that's because we'd have to specify an override variable if we're interested in overriding the default binary output path to something more traditional like bin for Go-based projects.

@timflannagan  if a user want to use some different path then `$GOPATH/bin` then they can set `$GOPATH` accordingly. I think that's how it is for go projects.

Reply:
Sounds good to me - updated.